test/suites/fips/validate-fips.robot

```diff
@@ -32,6 +32,9 @@ Verify Node RPMs FIPS Compliant
     [Documentation]    Performs a FIPS validation against the Installed RPMs
     Check Payload Tool Must Pass
 
+Verify Payload FIPS Compliant
```

Comment:
Nit: would using `Container Images` instead of `Payload` be more explicit?
The latter can be anything, so it was a bit confusing to read the code,